src/Service.Tests/SqlTests/SqlTestHelper.cs

```diff
@@ -96,6 +96,26 @@ public static void TestForErrorInGraphQLResponse(string response, string message
             }
         }
 
+        /// <summary>
+        /// Helper method to preprocess response to unescape unicode characters and remove all carriage returns/new lines.
+        /// </summary>
+        /// <param name="response">Raw response received from the API.</param>
+        /// <returns>Processed response containing unescaped unicode characters without new lines/carriage returns.</returns>
+        private static string PreprocessResponse(string response)
+        {
+            // Quote(") has to be treated differently than other unicode characters
+            // as it has to be added with a preceding backslash.
+            response = Regex.Replace(response, @"\\u0022", @"\\""");
```

Comment:
Better to use StringBuilder when doing replace performance wise.

Reply:
We cannot do regex replace on StringBuilders.